conf/cmi/user.role.content_producer.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   module:
     - content_translation
     - file
+    - group
```

Comment:
These need to be merged with the new roles as well.

Reply:
Checking and adding permissions regarding new roles.